charts/fybrik/files/taxonomy/taxonomy.json

```diff
@@ -26,6 +26,42 @@
     "Attribute": {
       "type": "string"
     },
+    "AttributeDefinition": {
+      "type": "object",
+      "required": [
+        "attribute",
+        "type"
+      ],
+      "properties": {
+        "attribute": {
+          "$ref": "#/definitions/Attribute",
+          "description": "Attribute name defined in the taxonomy"
+        },
+        "object": {
+          "$ref": "#/definitions/InstanceType",
+          "description": "A resource defined by the attribute (\"fybrikstorageaccount\",\"fybrikmodule\",\"cluster\")"
+        },
+        "objects": {
+          "description": "Argument types for cross-object attributes",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/InstanceType"
+          }
+        },
+        "scale": {
```

Comment:
Shouldn't scale, type, and unit be in a separate structure that can be reused by multiple InfraAttribute instances?

Reply:
Created a separate InfrastructureMetrics structure.